batch_processing.sh

```diff
@@ -58,14 +58,23 @@ start=$(date +%s)
 if [[ "$SCT_BP_DOWNLOAD" == "1" ]]; then
   sct_download_data -d sct_example_data
 fi
+
+# If there is no SCT_DIR set (i.e. this is a git clone), try to use the current directory instead
+if [[ -z "$SCT_DIR" ]]; then
+  SCT_DIR="$PWD"
+fi
+
+# Enter our data directory
 cd "$SCT_DIR/data/sct_example_data"
 
+# install the sct_deepseg model this script will use later
+sct_deepseg spinalcord -install
 
 # t2
 # ===========================================================================================
 cd t2
 # Segment spinal cord
-sct_deepseg_sc -i t2.nii.gz -c t2 -qc "$SCT_BP_QC_FOLDER"
+sct_deepseg spinalcord -i t2.nii.gz -qc "$SCT_BP_QC_FOLDER"
```

Comment:
Thank you for this! It's quite a radical change (hence the significant difference in results), but given that we've already migrated everything else over to `spinalcord`, I'm glad to have finally cleaned things up here as well. :)